Automattic/wp-calypso
# Purchases: Fix styling of chat button in auto-renwal dialog

Fixes #78379

Regression possibly introduced in #78173.

## Proposed Changes

* Ensure the `PrecancellationChatButton` has the correct, inline styles applied to it.

## Testing Instructions

The easiest is to hack the `auto-renew-disabling-dialog` like this:

```
diff --git a/client/me/purchases/manage-purchase/auto-renew-toggle/auto-renew-disabling-dialog/index.jsx b/client/me/purchases/manage-purchase/auto-renew-toggle/auto-renew-disabling-dialog/index.jsx
index 29f7f1bc61..c323106982 100644
--- a/client/me/purchases/manage-purchase/auto-renew-toggle/auto-renew-disabling-dialog/index.jsx
+++ b/client/me/purchases/manage-purchase/auto-renew-toggle/auto-renew-disabling-dialog/index.jsx
@@ -293,6 +293,7 @@ class AutoRenewDisablingDialog extends Component {
        render() {
                switch ( this.state.dialogType ) {
                        case DIALOG.GENERAL:
+                               return this.renderSurvey();
                                return this.renderGeneralDialog();
                        case DIALOG.ATOMIC:
                                return this.renderAtomicFollowUpDialog();
```

That way click on the `ON` link here on a purchase with auto-renew enabled will trigger the dialog:

<img width="275" alt="Screenshot 2023-06-19 at 15 17 40" src="https://github.com/Automattic/wp-calypso/assets/3392497/4e5a6a12-675c-4d2f-a8e1-2611a692259b">

The chat button should display properly in the dialog:

<img width="535" alt="Screenshot 2023-06-19 at 15 17 12" src="https://github.com/Automattic/wp-calypso/assets/3392497/da708084-14e0-40f5-b4dd-fb0c3ed6f6e6">

Make sure to test the cases from #78173 as well.
